tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -334,15 +342,18 @@ port::Status CudnnSupport::Init() {
     // Check whether loaded version of CuDNN matches what the source
     // was built with.
     size_t loaded_version = dynload::cudnnGetVersion();
-    bool library_loaded_matches_source = (loaded_version == CUDNN_VERSION);
+    size_t loaded_major_version = cudnnMajorVersion(loaded_version);
+    bool library_loaded_matches_source =
+        (loaded_major_version == cudnnMajorVersion(CUDNN_VERSION));
```

Comment:
So if I divide by CUDNN_VERSION by 100 instead of 1000 and rename to cudnnCompatibilityVersion, would that be sufficient?